tools/Replay.hpp

```diff
@@ -81,6 +85,31 @@ struct Histogram {
             std::cout << "< " << std::setw(4) << ms << " ms |" << std::string(chrs, '-') << "| " << _buckets[i] << "\n";
         }
     }
+
+    void dumpLatencyToCSV(std::string fileName)
```

Comment:
This is good - fine to have the data; but the graph is rather unexpected.

It seems we have two states - a slow and a fast one - sometimes we take >300ms to process things, and then we get many fewer tiles, and sometimes we are much quicker and get more high speed tiles. This points to an interesting potential problem.

We should log in a summary CSV - the number of tiles we got, and the number sub 10ms, the number up to 100ms, and the number over that: so four numbers:  #tiles, sub10ms, sub100ms, other

Hopefully we can come up with a single summary CSV that has our highest quality & most useful metrics in it =)